src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacPropertyAttributes/DacPropertyAttributesAnalyzer.cs

```diff
@@ -68,10 +67,48 @@ private static Task AnalyzePropertyAsync(SymbolAnalysisContext symbolContext, PX
 			if (!validSpecialTypes)
 				return;
 
+			CheckForPXDBCalcedAndUnboundTypeAttributes(symbolContext, pxContext, fieldAttributesRegister, property);
+
 			await CheckForFieldTypeAttributesAsync(property, symbolContext, pxContext, attributesWithInfos)
 					.ConfigureAwait(false);
 		}
 
+		private static void CheckForPXDBCalcedAndUnboundTypeAttributes(SymbolAnalysisContext symbolContext, PXContext pxContext,
+			FieldTypeAttributesRegister fieldAttributesRegister, IPropertySymbol propertySymbol)
+		{
+			symbolContext.CancellationToken.ThrowIfCancellationRequested();
+
+			var attributes = propertySymbol.GetAttributes();
+			var hasUnboundTypeAttribute = false;
+			var hasPXDBCalcedAttribute = false;
+
+			foreach (var attr in attributes)
```

Comment:
@metallmonster this is incomplete validation. It doesn't consider that some DAC property attributes could be derived from agregators which specify unbound type attribute. The whole point of the `GetFieldTypeAttributesInfos` method is to retrieve the attributes with the consideration for possible aggregators in inheritance chain

Reply:
ok